sched/mqueue/mqueue.h

```diff
@@ -53,12 +53,6 @@
 
 #define NUM_MSG_DESCRIPTORS  4
 
-/* This defines the number of messages to set aside for exclusive use by
- * interrupt handlers
- */
-
-#define NUM_INTERRUPT_MSGS   8
```

Comment:
It could be a good idea to include a fail-safe protection:

#if !defined(CONFIG_PREALLOC_MQ_IRQ_MSGS)
#define CONFIG_PREALLOC_MQ_IRQ_MSGS 8
#endif

Reply:
> It could be a good idea to include a fail-safe protection:
> 
> #if !defined(CONFIG_PREALLOC_MQ_IRQ_MSGS)
> #define CONFIG_PREALLOC_MQ_IRQ_MSGS 8
> #endif

We used to do these kinds of paranoid checks back in the old days when we hand edited config files.  This practice is not done anymore.  We trust the Kconfig files now.

Reply:
Yes, and since we always save the normalize(or smallest) defconfig now, it always require kconfig-frontend to expend config. And then all default value will bring back by the tools automatically.